packages/next/build/polyfills/object-assign.js

```diff
@@ -0,0 +1 @@
+export default Object.assign
```

Comment:
Can we make this `module.exports = Object.assign`? I think it'd save us a few bytes.

Reply:
Sync'd with @janicklas-ralph about this. Depending on hot it's imported, both are required.

Short term, the solution is this:

```js
module.exports = Object.assign;
module.exports.default = module.exports;
```

Long term, we can create a Resolve plugin that transforms the shim module depending on whether the requestor module imported as an ES Module or required it as CommonJS.

Reply:
Sure. I was looking into that too